src/test/java/domain/CarNameTest.java

```diff
@@ -10,10 +10,12 @@ class CarNameTest {
     @Test
     @DisplayName("최대 이름 길이보다 긴 이름으로 차 이름을 정하려고 할 때 에러")
     public void when_car_name_longer_than_max_length() {
+        //given
```

Comment:
```suggestion
        // given
```
다른 부분도 한 칸 씩 띄워주는게 가독성에서 좋은 것 같아요 ㅎㅎ 귀찮다면 다음 미션부터!